packaging/installer/kickstart-static64.sh

```diff
@@ -12,6 +12,10 @@
 #  --local-files              Use a manually provided tarball for the installation
 #  --allow-duplicate-install  do not bail if we detect a duplicate install
 #  --reinstall                if an existing install would be updated, reinstall instead
+#  --claim-token              specify a token to use for claiming the newly installed instance
```

Comment:
what I don't like here is different syntax for every script. we call claiming as:
`-token=hCdOx-dvi9Wf2A4RhUCAwI6XlJWddCoZAepekiG14ZDBZYAXfMZx0M3SrGeVPOLmzWa6UkaO9Ml7zceA-H6dPEr8Iqdeek3K-UYV3YOje35QQ2-oEpxK4aFv2ri2nHkJF5LqSKg -rooms=d31094b5-c7dd-409e-8dff-c5235e930dc1 -url=https://app.netdata.cloud`
e.g. `-token=XXX` but kickstart wants `--claim-token XXX` with space

Reply:
Then I strongly suggest we change the claiming script, not what I have here. The `=` syntax is actually more complicated to parse in shell script than not using it, the `claim-` prefix makes the meaning of the options in the installers unambiguous (assume you have no prior knowledge, what would you assume `proxy` to be for in an installation script?), and the double-dash lead-in is for consistency with the rest of the script, which we cannot change without breaking existing tooling users have made around working with these scripts.

Reply:
we would have to go in contact with Cloud guys as well:
- the Cloud UI generates the command for copy&pasting
- they would also have to handle the "if your agent version is older than this use this else another thing"

Although by default I trust your assessment of the version without `=` being better, changing it in Claiming script is too complicated (due to Cloud thing)

Reply:
Long-term though I think the goal is to not have users touch the claiming script except in specific exceptional cases. If we were to add a command for `netdatacli` to allow claiming of a running agent and merge this PR, we could pivot to not even mentioning the claiming script, and just point users at the installer options, the Docker environment variables, and the netdatacli command for claiming.

Reply:
I agree with @Ferroin on this one. I think that the claiming script should change not to require the `=` syntax.

Reply:
It must change anyway in my opinion, because '-token', '-url', '-rooms' are too generic as options.

They should be '-claim-token', '-claim-url' and '-claim-rooms' instead